src/index.html

```diff
@@ -6,9 +6,60 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+     crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"     rel="stylesheet">
+
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo_img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="index.html">APPLE</a>
```

Comment:
```suggestion
            <a class="nav__link is-active" href="#">Apple</a>
```
write all text in normal case and set for .nav__link in styles text-transform: uppercase